backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/document/DocBaseType.java

```diff
@@ -56,6 +56,8 @@ public enum DocBaseType implements ReferenceListAwareEnum
 	BillOfMaterialVersion(X_C_DocType.DOCBASETYPE_BOMFormula),
 	CostRevaluation(X_C_DocType.DOCBASETYPE_CostRevaluation),
 	ModularOrder(X_C_DocType.DOCBASETYPE_ModularOrder),
+	FinalInvoice(X_C_DocType.DOCBASETYPE_FinalPurchaseInvoice),
+	FinalCreditMemo(X_C_DocType.DOCBASETYPE_FinalCreditMemo),
```

Comment:
auch! why do we have to introduce new DocBaseTypes?!
This has a lot of implications... e.g. accounting, have u checked that works?!

I suggest using the standard invoice types and having specific sub-types

Reply:
thx, will use the standard invoice types and add specific sub-types